components/HomePage/LearnMore.js

```diff
@@ -87,4 +91,11 @@ const LearnMore = ({ className }) => {
   );
 };
 
+LearnMore.propTypes = {
```

Comment:
In a branch to be merged shortly, still pending review, I have disabled prop validation in eslint.

Reply:
Sounds good. Once that's completed I'd be happy to help clean up eslint warnings and add that to `npm test`.